handler.go

```diff
@@ -93,16 +127,48 @@ func (h *handler) Handle(_ context.Context, r slog.Record) error {
 	buf := newBuffer()
 	defer buf.Free()
 
+	buf.WriteString(ansiReset)
+
+	groups := []string{h.groups}
+
 	// write time
-	h.appendTime(buf, r.Time)
-	buf.WriteByte(' ')
+	a := slog.Time(slog.TimeKey, r.Time)
+	if h.replaceAttr != nil {
+		a = h.replaceAttr(groups, a)
```

Comment:
For the time, I believe you want to pass `nil` or `[]string{}` to `replaceAttr`. The message, source, level, and time attributes are outside of any groups, so they should get an empty set of groups when `replaceAttr` is called on them.

That way users can do tests like this to make sure that they replace a time attribute only for the whole record (rather than a time attribute in a sub-group):

```go
removeTime (groups []string, a slog.Attr) slog.Attr {
	if a.Key == slog.TimeKey && len(groups) == 0 {
		return slog.Attr{}
	}
	return a
}
```

Reply:
Yup. I think I picked it up [in slog's source, here](https://cs.opensource.google/go/x/exp/+/10a50721:slog/handler.go;l=264).